### PR TITLE
Story 3 - Claim a book

### PR DESCRIPTION
**Pull request by GDTimi**
Story 3 work complete, ready for review!

### CODE REVIEW

File: src/components/BookDetail/BookDetail.jsx [hunk 1]
```diff
@@ -36,12 +38,13 @@ function BookDetail() {
...
                     setBookGenre(bookData.data.genre.name)
                     setBookBlurb(bookData.data.blurb)
+                    setClaimedByName(bookData.data.claimed_by_name)
                 } else {
                     // Display an error message
                     setError(true)
                 }
             })
-    }, [id])
+    }, [id, claimedByName])
```
Comment (Geoff10) [inline src/components/BookDetail/BookDetail.jsx]: Why does the `useEffect` need claimedByName as a dependency?

File: src/components/BookDetail/ClaimBook/ClaimBook.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from "react"
```
Comment (Geoff10) [inline src/components/BookDetail/ClaimBook/ClaimBook.jsx]: It looks like you are importing the `useEffect` but using it.

**Review by Geoff10 — CHANGES_REQUESTED** [review]
Just one little comment and a question!

File: src/components/BookDetail/BookDetail.jsx (hunk 1 above)
Comment (henrymeager) [reply src/components/BookDetail/BookDetail.jsx]: we've removed claimedbyName from there now.

**Review by Geoff10 — APPROVED** [review]
Nice work!

**Pull request merged**